Mathlib/Data/Ordmap/Ordnode.lean

```diff
@@ -147,6 +146,27 @@ def empty : Ordnode α → Bool
   | node _ _ _ _ => false
 #align ordnode.empty Ordnode.empty
 
+-- porting note: workaround for leanprover/lean4#2049
```

Comment:
Are you actually using this? It should not be necessary if you use pattern matching, right?

Reply:
Huh... I introduced this to fix `noncomputable` issues for the decidability fixes I did, and I though it also allowed me to remove `noncomputable` for `alter` but it was my other `rec_eq_recC` hack for `Option` that fixed *that*.